ledger/sandbox/src/main/scala/com/digitalasset/platform/index/TransactionConversion.scala

```diff
@@ -119,9 +120,17 @@ object TransactionConversion {
         }
     ).filter(_.nonEmpty)
 
+  private def isCreateOrExercise(n: Node): Boolean = {
```

Comment:
No action to be taken here as well, but in the process of tightening the loop we are creating more and more convenience functions that could probably be made part of an allocation-free implicit value type (like `EventOps` and `TreeEventOps`) as they will come in handy along the way to keep the code tidy while not needlessly creating intermediate objects.

Reply:
Not sure what you're saying. This doesn't create an intermediary object though. Or do you mean we should keep this sort of convenience function in `EventOps`?

Reply:
Actually in this case the method doesn't operate on ledger api events, but on the daml lf transaction nodes.